tests/test_uta_loading.py

```diff
@@ -0,0 +1,108 @@
+import configparser
+import signal
+import unittest
+
+import sqlalchemy as sa
+import testing.postgresql
+
+import uta
+import uta.loading as ul
+import uta.models as usam
+
+
+class TestUtaLoading(unittest.TestCase):
+
+    def setUp(self):
+        self.db = testing.postgresql.Postgresql()
```

Comment:
I like this.

Reply:
Perhaps we need a base test class that can be reused across test classes. Or switch to pytest.